packages/catalog-model/src/entity/index.ts

```diff
@@ -17,6 +17,8 @@
 export { entityMetaGeneratedFields } from './Entity';
 export type { Entity, EntityMeta } from './Entity';
 export * from './policies';
+export { parseEntityName, parseEntityRef, serializeEntityRef } from './ref';
+export type { EntityRefContext } from './ref';
```

Comment:
Should not export this imo

Reply:
Since it's used in another exported API, it should by extension be exported too, right? I mean, it may not be common to have to refer to it by name before passing it into parseEntityRef or so, but still

Reply:
Not necessarily, we'd export it to make it more convenient if you want to pass this structure around in external code, but that code might as well pass objects of similar shape around. If it grows more complex it may make sense to export, but imo we should be quite conservative in the amount of things we export

Reply:
i'mma stop exporting it!